src/components/dialogs/ResolutionDateDialog.vue

```diff
@@ -62,6 +66,10 @@ export default class ResolutionDateDialog extends Vue {
     return this.date ? '' : this.errorMsg
   }
 
+  get currentDateString (): string {
+    return this.dateToDateString(new Date())
```

Comment:
Again, use the current date that the app maintains, not `new Date()`.

Reply:
Fixed 👍